test/meshfiles/geojson/sample_chicago_buildings.geojson

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This is a general comment about all the files this PR adds, not only this one: There are several folders and files of different extensions, which many of our developers and users might not be familiar with. For future developer's reference, would it make sense to write a README under each of the `shp` and `geojson` folders that would explain some things such as:
- What is each file extension (.geojson, .cpg, dbf, .shp, .shx) for?
- What are groups of files (5poly, multipoly, etc.) for?

Reply:
done.